NDB.Covid19/NDB.Covid19.Droid/Views/AuthenticationFlow/CountriesConsentActivity.cs

```diff
@@ -67,19 +66,10 @@ private void InitLayout()
             _closeButton.ContentDescription = CLOSE_BUTTON_ACCESSIBILITY_LABEL;
 
             //Button click events
-            _closeButton.Click += new StressUtils.SingleClick((sender, e) => ShowAreYouSureToExitDialog(), 500).Run;
-            _consentButton.Click += new StressUtils.SingleClick(async (o, args) =>
+            _closeButton.Click += new StressUtils.SingleClick((sender, e) => ShowAreYouSureToDenyConsentDialog(), 500).Run;
```

Comment:
i do not understand.. is this really requested change? this is totally not user friendly. close button is always understood as leaving the whole flow, and now user will be redirected to the next page? @Frederik-S-N-dk

Reply:
@nc-kano I'll ask FHI again. But at least that is what we agreed upon on the last meeting.
So wait with merge of this PR until I have clarified with them again

Reply:
@Frederik-S-N-dk ok, user must be aware that this is totally not user friendly and misleading to the user. as i remember there is also a requirement in Android review process that we should not change the default button behaviors such as back/close/next/arrow buttons.
<img width="878" alt="image" src="https://user-images.githubusercontent.com/49269988/103648379-1488e200-4f5d-11eb-981d-e82ba74224ca.png">